src/Symfony/Component/Yaml/Inline.php

```diff
@@ -562,6 +562,8 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         }
 
                         return;
+                    case 0 === strpos($scalar, '!php/const:'):
+                        return @constant(substr($scalar, 11));
```

Comment:
I guess getting `null` should be expected (a constant could be defined in an environment and not in another), the flag let the user decide it in a known and explicit way.